src/MarkdownTextInput.web.tsx

```diff
@@ -438,8 +438,14 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
         setEventProps(e);
         if (divRef.current) {
           const valueLength = value ? value.length : 0;
-          CursorUtils.setCursorPosition(divRef.current, contentSelection.current ? contentSelection.current.end : valueLength);
-          updateSelection(event);
+          let startPosition = valueLength;
+          let endPosition = null;
+          if (contentSelection.current) {
+            startPosition = contentSelection.current.start;
+            endPosition = contentSelection.current.end;
+          }
+          CursorUtils.setCursorPosition(divRef.current, startPosition, endPosition);
```

Comment:
I don't like that we create two non-const variables here.

Suggestion 1:
```suggestion
          if (contentSelection.current) {
            CursorUtils.setCursorPosition(divRef.current, contentSelection.current.start, contentSelection.current.end);
          } else {
            CursorUtils.setCursorPosition(divRef.current, valueLength, null);
          }
```

Suggestion 2:
```suggestion
          const [startPosition, endPosition] = contentSelection.current
            ? [contentSelection.current.start, contentSelection.current.end]
            : [valueLength, null];
          CursorUtils.setCursorPosition(divRef.current, startPosition, endPosition);
```

Suggestion 3:
```suggestion
          CursorUtils.setCursorPosition(
            divRef.current,
            contentSelection.current ? contentSelection.current.start : valueLength,
            contentSelection.current ? contentSelection.current.end : null);
```

Reply:
I like suggestion number 1, thanks